### ISSUE TITLE
CcrRepositoryIT can exceed suite timeout.

### ISSUE THREAD

**Issue by jtibshirani** [elastic/elasticsearch#64372]
**Build scan**: https://gradle-enterprise.elastic.co/s/yyzru5yqee24k

**Repro line**:
```
./gradlew ':x-pack:plugin:ccr:internalClusterTest' --tests "org.elasticsearch.xpack.ccr.CcrRepositoryIT.testIndividualActionsTimeout" \
  -Dtests.seed=DEAD8C0434DD4D9C \
  -Dtests.security.manager=true \
  -Dtests.locale=es-SV \
  -Dtests.timezone=Europe/San_Marino \
  -Druntime.java=8
```

**Reproduces locally?**: no.

**Applicable branches**: 7.10, likely others

**Failure history**: there have been 4 failures in the past month, mostly in PR builds.

**Failure excerpt**:
```
org.elasticsearch.xpack.ccr.CcrRepositoryIT > testIndividualActionsTimeout FAILED
    java.lang.Exception: Test abandoned because suite timeout was reached.
        at __randomizedtesting.SeedInfo.seed([DEAD8C0434DD4D9C]:0)

org.elasticsearch.xpack.ccr.CcrRepositoryIT > classMethod FAILED
    java.lang.Exception: Suite timeout exceeded (>= 1200000 msec).
        at __randomizedtesting.SeedInfo.seed([DEAD8C0434DD4D9C]:0)
```

**Comment by dimitris-athanasiou** [comment]
Another failure in https://gradle-enterprise.elastic.co/s/nb4fyn5ppw2vu

**Comment by tlrx** [comment]
Thanks! I'm on it.

**Comment by tlrx** [comment]
This has been fixed by #66008
